apps/web/src/components/Notifications/notifications/NotificationPostPreview.tsx

```diff
@@ -0,0 +1,62 @@
+import { useFragment } from 'react-relay';
```

Comment:
a new, completely self-contained component that can be dropped in anywhere we want to display an NFT preview within a notification (e.g. someone admired your post, someone commented on your post)

Reply:
how come it's tied to a Post? would it be possible to use this for Mint notifications, or in the future "X admired your item" notifications?

Reply:
Absolutely can!!

Didn't use it for mint notifications because that had a quirk of being able to show multiple mints. Didn't feel as high prio but can follow up!